rose/server/game.py

```diff
@@ -56,6 +56,7 @@ def stop(self):
 
     def add_player(self, name):
         if name in self.players:
+            print 'Failed to add player:', name, 'player already exists'
             raise error.PlayerExists(name)
```

Comment:
We should make sure this error has clear text when printed, so we can do later:

    except Exception as e:
        print "Error handling message: %s" % e